lib/utils/containsString.js

```diff
@@ -28,15 +30,15 @@ module.exports = function containsString(
 };
 
 function testAgainstString(value, comparison) {
-	if (!comparison) return false;
-
-	if (comparison[0] === '/' && comparison[comparison.length - 1] === '/') {
+	if (
+		!comparison ||
+		_.isRegExp(comparison) ||
+		(comparison.startsWith('/') && comparison.endsWith('/'))
+	) {
```

Comment:
Let's split it intro three `if` (like it was before). And use `typeof comparison !== 'string'` instead checking for RegEx.

Reply:
Not sure if that will be sufficient, but I will try